x-pack/plugin/snapshot-repo-test-kit/src/main/java/org/elasticsearch/repositories/blobstore/testkit/RegisterAnalyzeAction.java

```diff
@@ -250,4 +260,32 @@ public String getDescription() {
             );
         }
     }
+
+    static long longFromBytes(BytesReference bytesReference) {
+        if (bytesReference.length() == 0) {
+            return 0L;
+        } else if (bytesReference.length() == Long.BYTES) {
+            try (var baos = new ByteArrayOutputStream(Long.BYTES)) {
```

Comment:
Don't we have some library functions to deal with this directly?

Reply:
Apparently not, or at least I couldn't find any.